calendar-bundle/composer.json

```
@@ -19,12 +19,12 @@
         "contao/core-bundle": "self.version",
         "friendsofsymfony/http-cache": "^2.4",
         "patchwork/utf8": "^1.2",
-        "symfony/config": "4.4.* || 5.2.*",
-        "symfony/dependency-injection": "4.4.* || 5.2.*",
+        "symfony/config": "4.4.* || 5.3.*",
```

```suggestion
        "symfony/config": "4.4.* || 5.2.* || 5.3.*",
```

There is no reason to drop Symfony 5.2 support afaik?

---

I don’t see a reason to support Symfony 5.2 as it is EOL. Why don’t we support 5.0 and 5.1?

---

Symfony 5 was introduced with Contao `4.11.0`, starting with Symfony 5.2, since that was the active Symfony version at the time. `5.2.*` should definitely stay supported, otherwise a downgrade path for Symfony to 4.4 is necessary in some circumstances.

---

But this pull request is for Contao `4.12.0` can’t we move up the dependecy to `5.3.*` the same way we moved it up from `4.3.*` to `4.4.*` in Contao `4.9`?

---

Running Symfony 5.2 isn't a viable option, same as Contao 4.11 (as soon as 4.12 is released), is it? All extensions blocking the update have the same problem IMHO.

I think we should consider using open requirements (`^` instead of `.*`) again in the future, though.

---

I think we made the decision to use `X.X.*` before we had the `contao/conflicts` package. With `contao/conflicts` we can IMO move back to using `^X.X` for the Symfony packages because if incompatibilities occur with a new Symfony version we can quickly add conflicts for the affected packages.

@contao/developers WDYT?

I would update this PR then to use `^5.2`.

---

> But this pull request is for Contao `4.12.0` can’t we move up the dependecy to `5.3.*` the same way we moved it up from `4.3.*` to `4.4.*` in Contao `4.9`?

It's different there as `4.4.*` is the minimum requirement - and still is in Contao `4.12.0` and `4.13.0`. Only an upgrade path would be allowed, not a downgrade path. However a downgrade because of this is probably more an edge case as I don't expect many other dependencies to also limit the Symfony version to a bugfix version.

---

> I would update this PR then to use `^5.2`.

Done in cfafcf4995665f400cb5c8c4428ff4823ba4ed46